src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementForm.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useCallback } from 'react';
+import { defineMessages } from 'react-intl';
+import { ColonyRole, ROOT_DOMAIN_ID } from '@colony/colony-js';
+import sortBy from 'lodash/sortBy';
+
+import { Address } from '~types/index';
+import { DomainFieldsFragment } from '~data/generated';
+import { InputLabel, Textarea, Select } from '~core/Fields';
+
+import PermissionManagementCheckbox from './PermissionManagementCheckbox';
+import { availableRoles } from './constants';
+
+import styles from './PermissionManagementDialog.css';
+
+const MSG = defineMessages({
+  domain: {
+    id: 'dashboard.PermissionManagementDialog.PermissionManagementForm.domain',
+    defaultMessage: 'Domain',
+  },
+  permissionsLabel: {
+    id: `dashboard
+      .PermissionManagementDialog.PermissionManagementForm.permissionsLabel`,
+    defaultMessage: 'Permissions',
+  },
+  reason: {
+    id: 'dashboard.PermissionManagementDialog.PermissionManagementForm.reason',
+    defaultMessage: 'Explain why you’re creating this motion (optional)',
```

Comment:
Please change this to: 
> Explain why you’re making these changes (optional)

_(And yes I know this wasn't in the original spec. Sorry about this)_